io_scene_godot/converters/multimesh.py

```diff
@@ -0,0 +1,176 @@
+import math
+import bpy
+import mathutils
+
+from ..structures import (
+    Array, NodeTemplate, InternalResource, fix_matrix, mat4_to_string)
+from .mesh import ArrayMeshResourceExporter
+
+
+def export_multimesh_node(escn_file, export_settings,
+                          obj, parent_gd_node):
+    """Export blender particle to a MultiMeshInstance"""
+    context = bpy.context
+    dg = context.evaluated_depsgraph_get()
+    ob = context.object.evaluated_get(dg)
+
+    multimeshid_active=None
+    for i in ob.particle_systems:

```

Comment:
maybe use `for ps in ob.particle_systems` here?  which is more concise.

Reply:
you are right!